### PR TITLE
AT_03.01.08LinkRESTAPIchangesColor

### PR DESCRIPTION
**Pull request by AlisaKapitanova**
https://trello.com/c/o8zt2uCC/2033-ac030108-footer-link-rest-api-changes-color

### CODE REVIEW

**Review by MariiaHolinko — APPROVED** [review]

**Review by NatalyaVasylyeva — APPROVED** [review]

File: cypress/e2e/tests/headerAndFooter.cy.js [hunk 1]
```diff
@@ -276,4 +276,10 @@ describe('headerAndFooter', () => {
           .should('have.text', resultSearchBoxData.resultSearchNoMatchMsg);
         })
     });
+
+    it('AT_03.01.08 Link REST API changes color', () => {
+        headerAndFooter
+            .realHoverRestAPILink()
+            .getRestAPILink().should('have.css', 'color', 'rgb(5, 135, 212)')
```
Comment (Rom-T) [inline cypress/e2e/tests/headerAndFooter.cy.js:283]: Please move `rgb(5, 135, 212)` in the fixtures. And move .should to the next line

File: cypress/pageObjects/HeaderAndFooter.js [hunk 2]
```diff
@@ -158,6 +158,11 @@ class HeaderAndFooter {
         this.getSearchBoxInputField().type('{enter}');
         return new BuiltInNodePage ();
         }
+
+    realHoverRestAPILink() {
```
Comment (Rom-T) [inline cypress/pageObjects/HeaderAndFooter.js:162]: Please rename method to `hoverRestAPILink()`

File: cypress/e2e/tests/headerAndFooter.cy.js (hunk 1 above)
Comment (AlisaKapitanova) [reply cypress/e2e/tests/headerAndFooter.cy.js]: done

File: cypress/pageObjects/HeaderAndFooter.js (hunk 2 above)
Comment (AlisaKapitanova) [reply cypress/pageObjects/HeaderAndFooter.js]: done, thank you for review

**Review by DimaCurkanu — COMMENTED** [review]
Please, delete 'only'

**Pull request merged**